.travis.yml

```diff
@@ -1,9 +1,23 @@
 language: vim
+rvm:
+  - 1.8.7 # Use stock
+  - 1.9.2 # Build latest
```

Comment:
@cirosantilli Interesting, hadn't read up on how extensively you can customize the build matrix, even exclusions! Neat.